src/builtins/Transformers.jl

```diff
@@ -343,30 +343,50 @@ otherwise, ignore specified features.
     │ 3   │ 1.14708   │ 3     │
 
 """
-@with_kw_noshow mutable struct Standardizer <: Unsupervised
-    features::Vector{Symbol} = Symbol[] # features to be standardized; empty means all
-    ignore::Vector{Symbol} = Symbol[] # features to be ignored
+mutable struct Standardizer <: Unsupervised
+    features::Vector{Symbol} # features to be standardized; empty means all
+    ignore::Bool # features to be ignored
+    ordered_factor::Bool
+    count::Bool
+end
+
+function MLJBase.clean!(transformer::Standardizer)
+    err = ""
+    if isempty(transformer.features) && transformer.ignore
+        err *= "Features to be ignored must be specified in features field."
+    end
+    return err
+end
+
+# keyword constructor
+function Standardizer(
+    ; features=Symbol[], ignore=false, ordered_factor=false, count=false)
+    transformer = Standardizer(features, ignore, ordered_factor, count)
+    message = MLJBase.clean!(transformer)
+    isempty(message) || throw(ArgumentError(message))
+    return transformer
 end
 
 function MLJBase.fit(transformer::Standardizer, verbosity::Int, X)
     all_features = Tables.schema(X).names
     mach_types   = collect(eltype(selectcols(X, c)) for c in all_features)
+    type_tup = [
+        (transformer.ordered_factor, Union{Continuous, OrderedFactor}),
+        (transformer.count, Count)
+    ]
+    additional_types = Union{last.(filter!(x->x[1], type_tup))...}
 
-    issubset(transformer.ignore, all_features) ||
-        @warn "Some ignored features not present in table to be fit. "
     # determine indices of all_features to be transformed
     if isempty(transformer.features)
         cols_to_fit = filter!(eachindex(all_features) |> collect) do j
-            !(all_features[j] in transformer.ignore) &&
-                mach_types[j] <: AbstractFloat
+            mach_types[j] <: Union{AbstractFloat, additional_types}
```

Comment:
The condition we want here is now `scitypes[j] <: AllowedScitype`, and similarly later on.